stbt/__init__.py.in

```diff
@@ -33,6 +35,7 @@
     UITestError, \
     UITestFailure, \
     wait_until
+del sys.path[0]
```

Comment:
Rather than renaming this file I think I'd prefer a forwarding module that just contains

    import sys
    sys.path.insert(0, "@LIBEXECDIR@/stbt")
    from stbt import *
    del sys.path[0]

That way this is still a `.py` file in the source repo so tools will play nicer with it, e.g. nosetests, syntax highlighting.